### DISCUSSION TITLE
Source Type specification from CLI

### DISCUSSION THREAD

**Discussion by azizbodal** [comunica/comunica#963] [Q&A]
Hi,
Is there a way to specify the source type when using the Comunica CLI? I am using GraphDB and I believe the CLI is inferring the wrong source type (I get a 400 error). I am able to get it to work via the JavaScript library when I set the type to 'sparql'.

Thanks!

**Comment by jeswr** [comment]
I believe a similar question was just answered in https://github.com/comunica/comunica/issues/962

**Comment by azizbodal** [comment]
Great thanks!
